spec/controllers/dashboard/firms_controller_spec.rb

```diff
@@ -0,0 +1,87 @@
+module Dashboard
+  RSpec.describe Dashboard::FirmsController, type: :controller do
+    let(:principal) { FactoryGirl.create(:principal) }
```

Comment:
Most of the places are using just create. It is better to follow this convention.